packages/react/src/SubNav/SubNav.test.tsx

```diff
@@ -125,24 +141,28 @@ describe('SubNav', () => {
     expect(results).toHaveNoViolations()
   })
 
+  it('does not hide submenu items on narrow viewports', async () => {
+    const {getByRole} = render(<MockSubNavFixtureWithSubMenu />)
+
+    const buttonEl = getByRole('button', {name: 'Navigation menu. Current page: Copilot'})
+
+    await userEvent.click(buttonEl)
+
+    expect(getByRole('link', {name: 'Copilot feature page one'})).toBeInTheDocument()
+  })
+
+  it('hides collapsed submenu items on large viewports', async () => {
+    mockUseWindowSize.mockImplementation(() => ({isLarge: true}))
+
+    const {queryByRole} = render(<MockSubNavFixtureWithSubMenu />)
+
+    expect(queryByRole('link', {name: 'Copilot feature page one'})).not.toBeInTheDocument()
```

Comment:
Can we also check that the submenu appears when parent item is hovered? Assuming it's not already covered in a different test.

Reply:
It's implicitly tested as part of the test here

https://github.com/primer/brand/blob/8c3c2a270dd96adf0af866584080e9246327e975/packages/react/src/SubNav/SubNav.test.tsx#L196-L208